gcam_to_tell/gcam-usa_to_tell.py

```diff
@@ -22,9 +22,11 @@ def get_gcam_electricity_load(
   # run the load query; note that we're assuming it's the first query in the file
   load = conn.runQuery(queries[0])
 
+  # TODO update the scenario field to use official IM3 scenario names
+
   # aggregate the load
   aggregated = load[
-    (load.input == 'electricity domestic supply') &
+    (load.input.str.startswith('electricity domestic supply'))
```

Comment:
The query being used is pulling way more data than needed.  Maybe at small scales this isn't a big deal.  But in case it is of concern, you do the filtering on the query side instead of the python side.  Which of course reduces the volume of data coming out of the XMLDB and into your python script.

The query would then become:
```
<supplyDemandQuery title="inputs by tech">
    <axis1 name="input">input</axis1>
    <axis2 name="Year">demand-physical[@vintage]</axis2>
    <xPath buildList="true" dataName="input" group="false" sumAll="false">
        *[@type='sector']/*[@type='subsector']/*[@type='technology']/*[@type='input' and starts-with(@name, 'electricity domestic supply')]/demand-physical/node()
      </xPath>
    <comments/>
</supplyDemandQuery>
```

And to take it even further, we could skip having to aggregate sectors (and segments) with:
```
<supplyDemandQuery title="inputs by tech">
    <axis1 name="input">input</axis1>
    <axis2 name="Year">demand-physical[@vintage]</axis2>
    <xPath buildList="true" dataName="input" group="false" sumAll="true">
        *[@type='sector' (: collapse :)]/*[@type='subsector' (: collapse :)]/*[@type='technology' (: collapse :)]/*[@type='input' and starts-with(@name, 'electricity domestic supply')]/demand-physical/node()
      </xPath>
    <comments/>
</supplyDemandQuery>
```